ddtrace/contrib/internal/openai/utils.py

```diff
@@ -51,6 +55,14 @@ def __iter__(self):
         exception_raised = False
         try:
             for chunk in self.__wrapped__:
+                # Handle both completion and response types
+                if hasattr(chunk, "type") and chunk.type == "response.completed":
+                    _tag_tool_calls(self._dd_integration, self._dd_span, chunk.response)
+                    if hasattr(chunk.response, "id"):
+                        self._dd_span.set_tag("openai.response.id", chunk.response.id)
+                    if hasattr(chunk.response, "created_at"):
+                        self._dd_span.set_tag_str("openai.response.created_at", str(chunk.response.created_at))
+                    self._dd_integration.record_usage(self._dd_span, chunk.response.usage)
```

Comment:
Can we remove this since `_process_finished_stream()` already includes a call to `_set_token_metrics()`?